sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+friendly_name: Feature Usage Metrics
+description: |-
+  Metrics from metrics pings for mobile feature usage dashboards (iOS)
+owners:
+- rzhao@mozilla.com
+labels:
+  incremental: true
+  owner1: rzhao
+  dag: bqetl_default
+scheduling:
+  dag_name: bqetl_mobile_feature_usage
+  task_name: ios_feature_usage_metrics__v1
```

Comment:
You could omit the task_name here, the query file path would be used to infer the task name.